server/modules/collectImport/service/collectImport/collectImportJob.js

```diff
@@ -8,7 +8,6 @@ const CategoriesImportJob = require('./metaImportJobs/categoriesImportJob')
 const TaxonomiesImportJob = require('./metaImportJobs/taxonomiesImportJob')
 const NodeDefsImportJob = require('./metaImportJobs/nodeDefsImportJob')
 const RecordsImportJob = require('./dataImportJobs/recordsImportJob')
-const SurveyIndexGeneratorJob = require('../../../survey/service/surveyIndexGeneratorJob')
 const SurveyRdbGeneratorJob = require('../../../surveyRdb/service/surveyRdbGeneratorJob')
 const RecordCheckJob = require('../../../survey/service/recordCheckJob')
```

Comment:
RecordCheckJob import is unused, should it be commented out until RecordCheckJob is restored?

Reply:
yes it's temporary left there. [there's another task to get rid of it in collectImportJob](https://trello.com/c/Fk8LbB0F/185-refactor-collectimportjob-surveypublishjob-recordcheckjob) .